Ginger/GingerCoreCommon/VariablesLib/VariableBase.cs

```diff
@@ -367,10 +367,10 @@ public static void GetListOfUsedVariables(object item, ref List<string> usedVari
                                             if (usedVariables.Contains(value.ToString()) == false)
                                                 usedVariables.Add(value.ToString());
                                     }
-                                }
-                                else if(mi.Name == "ValueCalculated" && mi.DeclaringType.Name == "FlowControl") // get used variable in flow control with set variable action type.
-                                {
-                                    string[] vals = ((string)value).Split(new[] { '=' });
+                                }                               
+                                else if(mi.Name == "FlowControlAction" && value.ToString() == "SetVariableValue") //get used variable in flow control with set variable action type.
+                                {                                    
```

Comment:
i'll approve it but never use String based comparison, but use enum or nameof() to compare with, like:
if(mi.Name == nameof(FlowControl.FlowControlAction) && value.ToString() == FlowControl.eFlowControlAction.SetVariableValue)